Unable to run metabase built from source

I'm getting the following error while running metabase.jar built from latest master code
v0.23.0-snapshot 0aa2df9 master 2017-02-22'

```02-24 04:21:25 INFO metabase.util :: Loading Metabase...
02-24 04:21:25 INFO util.encryption :: DB details encryption is DISABLED for this Metabase instance. ?
02-24 04:21:29 INFO metabase.core :: Starting Metabase in STANDALONE mode
02-24 04:21:29 INFO metabase.core :: Launching Embedded Jetty Webserver with config:
 {:port 3000}

02-24 04:21:29 INFO metabase.core :: Starting Metabase version v0.23.0-snapshot (0aa2df9 master) ...
02-24 04:21:29 INFO metabase.core :: System timezone is 'America/New_York' ...
02-24 04:21:29 DEBUG metabase.driver :: Registered driver :mysql ?
02-24 04:21:29 DEBUG metabase.driver :: Registered driver :sqlite ?
02-24 04:21:30 DEBUG metabase.driver :: Registered driver :mongo ?
02-24 04:21:30 DEBUG metabase.driver :: Registered driver :druid ?
java.lang.NoClassDefFoundError: metabase/driver/bigquery$?s__GT_str
	at metabase.driver.bigquery__init.load(Unknown Source)
	at metabase.driver.bigquery__init.<clinit>(Unknown Source)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at clojure.lang.RT.classForName(RT.java:2168)
	at clojure.lang.RT.classForName(RT.java:2177)
	at clojure.lang.RT.loadClassForName(RT.java:2196)
	at clojure.lang.RT.load(RT.java:443)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5774)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at metabase.driver$find_and_load_drivers_BANG_.invokeStatic(driver.clj:284)
	at metabase.driver$find_and_load_drivers_BANG_.invoke(driver.clj:278)
	at metabase.core$init_BANG_.invokeStatic(core.clj:111)
	at metabase.core$init_BANG_.invoke(core.clj:95)
	at metabase.core$start_normally.invokeStatic(core.clj:198)
	at metabase.core$start_normally.invoke(core.clj:192)
	at metabase.core$_main.invokeStatic(core.clj:290)
	at metabase.core$_main.doInvoke(core.clj:285)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at metabase.core.main(Unknown Source)
Caused by: java.lang.ClassNotFoundException: metabase.driver.bigquery$?s__GT_str
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 39 more
02-24 04:21:30 ERROR metabase.core :: Metabase Initialization FAILED:  metabase/driver/bigquery$?s__GT_str
02-24 04:21:30 INFO metabase.core :: Metabase Shutting Down ...
02-24 04:21:30 INFO metabase.core :: Metabase Shutdown COMPLETE
```

I'm able to run 0.22 release but I wanted to tinker with standardSql in Big query

**Linux Distribution**
```
NAME="Amazon Linux AMI"
VERSION="2015.03"
ID="amzn"
ID_LIKE="rhel fedora"
VERSION_ID="2015.03"
PRETTY_NAME="Amazon Linux AMI 2015.03"
ANSI_COLOR="0;33"
CPE_NAME="cpe:/o:amazon:linux:2015.03:ga"
HOME_URL="http://aws.amazon.com/amazon-linux-ami/"
Amazon Linux AMI release 2015.03
```

**Java (Oracle)**
```
java version "1.8.0_121"
Java(TM) SE Runtime Environment (build 1.8.0_121-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)
```

**Yarn**
```
0.20.3
```

**Lein**
```
Leiningen 2.7.1 on Java 1.8.0_121 Java HotSpot(TM) 64-Bit Server VM
```

--- a/src/metabase/driver/bigquery.clj
+++ b/src/metabase/driver/bigquery.clj
@@ -210,15 +210,15 @@
 (defn- date-add [unit timestamp interval]
   (hsql/call :date_add timestamp interval (hx/literal unit)))
 
-;; µs = unix timestamp in microseconds. Most BigQuery functions like strftime require timestamps in this format
+;; microseconds = unix timestamp in microseconds. Most BigQuery functions like strftime require timestamps in this format
 
-(def ^:private ->µs (partial hsql/call :timestamp_to_usec))
+(def ^:private ->microseconds (partial hsql/call :timestamp_to_usec))
 
-(defn- µs->str [format-str µs]
+(defn- microseconds->str [format-str µs]
   (hsql/call :strftime_utc_usec µs (hx/literal format-str)))
 
 (defn- trunc-with-format [format-str timestamp]
-  (hx/->timestamp (µs->str format-str (->µs timestamp))))
+  (hx/->timestamp (microseconds->str format-str (->microseconds timestamp))))
 
 (defn- date [unit expr]
   {:pre [expr]}

